src/hotspot/share/gc/shared/gcCause.hpp

```diff
@@ -117,8 +115,7 @@ class GCCause : public AllStatic {
   // Causes for collection of the young generation
   inline static bool is_allocation_failure_gc(GCCause::Cause cause) {
     // _allocation_failure is the generic cause a collection for allocation failure
-    return (cause == GCCause::_allocation_failure ||
-            cause == GCCause::_shenandoah_allocation_failure_evac);
+    return cause == GCCause::_allocation_failure;
```

Comment:
Shenandoah does not use this method. Nobody else needs to know about this Shenandoah specific value.